tests/context.py

```
@@ -17,13 +17,12 @@
 # limitations under the License.
 
 
-__module__ = """tests.context"""
+__module__ = """tests"""
 """This is pythonrepo testing module Template."""
 
 
-__module__ = """tests"""
+__name__ = f"""{module}.context"""  # skipcq: PYL-W0622
```

Ruff (0.12.2)

Undefined name `module`

(F821)